NuGet.Config

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+<!--
+    This configuration tells the .NET Core SDK to build IQ# using the
```

Comment:
Technically, it just tells .net core to also look in the `qdk-alpha` feed, not to build against using that feed. Each project then specifies which version of the QDK to build against.